### ISSUE TITLE
Update to use pyasl.ExoplanetEU2()

### ISSUE THREAD

**Issue by jason-neal** [DanielAndreasen/SWEETer-Cat#31]
`pyasl.ExoplanetEU2` is now considered the correct version to use, see [here](http://www.hs.uni-hamburg.de/DE/Ins/Per/Czesla/PyA/PyA/pyaslDoc/resBasedDoc/exoplanetEU.html)

- Downloads as VOTable instead of scv
- Access to all the data, columns visible by doing
```
from PyAstronomy import pyasl
v = pyasl.ExoplanetEU2()
v.showAvailableData()
```

### Issues
- Most parameter names change, so will need updated. e.g. `plName -> name`, `plRadius -> radius`
Or
- Have a identification dict like what is in `pyasl.exoplanetEU()` [here](https://github.com/sczesla/PyAstronomy/blob/master/src/pyasl/resBased/exoplanetEU.py#L212)
```
# Identify exoplanet.eu csv column names with internal column names
self._ident = {"name":"plName", "mass":"plMass", "radius":"plRadius",  ...
```

**Comment by DanielAndreasen** [comment]
Just to have the code ready for a pandas DataFrame:

```
from PyAstronomy import pyasl
v = pyasl.ExoplanetEU2()
df = v.getAllDataPandas()
```

Regarding the new names in the `ExoplanetEU2` version, I suggest we update our code rather than using a dictionary to rename. But there might be some unforseen conflicts along the way.

**Comment by jason-neal** [comment]
So currently the dictionary is still used to turn the new exoplanetEU2 names into the names we had.

I had a strange error in which I was getting "plName" not found. In  the
 `show_planet = bool(~d['plName'].isnull().values[0])` of `stardetail` 
I only managed to reproduce it twice (once in master). 
I don't know if this was a issue with caching though instead. (That's the only thing  I can think off, as it resolved itself when I tried to print the columns of the dataframe and refreshed the app.)
